android/src/main/java/ch/milosz/reactnative/RNZoomUsModule.java

```diff
@@ -281,6 +332,287 @@ public void connectAudio() {
     connectAudioWithVoIP();
   }
 
+  @ReactMethod
+  public void isMeetingConnected(Promise promise) {
+    WritableArray rnUserList = Arguments.createArray();
+    ZoomSDK zoomSDK = ZoomSDK.getInstance();
+
+    if (!zoomSDK.isInitialized()) {
+      promise.resolve(false);
+      return;
+    }
+
+    final InMeetingService inMeetingService = zoomSDK.getInMeetingService();
+
+    promise.resolve(inMeetingService.isMeetingConnected());
+  }
+
+  @ReactMethod
+  public void isMeetingHost(Promise promise) {
+    WritableArray rnUserList = Arguments.createArray();
+    ZoomSDK zoomSDK = ZoomSDK.getInstance();
+
+    if (!zoomSDK.isInitialized()) {
+      promise.resolve(false);
+      return;
+    }
+
+    final InMeetingService inMeetingService = zoomSDK.getInMeetingService();
+
+    promise.resolve(inMeetingService.isMeetingHost());
+  }
+
+  @ReactMethod
+  public void getInMeetingUserList(Promise promise) {
```

Comment:
you are fetching user ids, not users
```suggestion
  public void getInMeetingUserIdList(Promise promise) {
```